test_runner.py

```diff
@@ -265,11 +265,14 @@ def build_test_list():
         ),
         OverrideDefinitions(
             [
-                ["--estimate.memory=True", "--estimate.mode=real"],
+                [
+                    "--memory_estimation.enabled",
+                    "--memory_estimation.fake_mode_only",
+                ]
             ],
             "FSDP2 Memory Tracking and Estimation",
             "fsdp2_mem_tracker",
-            ngpu=4,
+            ngpu=8,
```

Comment:
Why do we need 8 GPUs?

Reply:
The tests for 4 GPUs are already taking a lot of CI time, so I thought shifting some workload to 8 might be better

Reply:
Currently 8 gpu test is only used for 3D parallelism test. Not sure if we should use it for other purposes. @wconstab for context.

IMO before we understand & agree on whether we should do "load balancing" between the two, we should use 4-gpu test for everything other than those necessitate 8 gpus.